Hairtrigger-Bunnies/nodejs-bangazon-site
# El user acct settings

Changed:
- Added error handler to app.js
- Added pug view for user account settings and link in nav
- Added function to get user's details
- Added jQuery to make it easier to hide/show the form to edit user details

Because:
- For ticket #17 , user can view and edit their account info
- error handler because it was not included in the boilerplate although the directions to make one were

To test:
(steps to test -- include everything!)
1. pull branch
1. run `npm install` then `npm start`
1. go to your localhost and log in, then click "my account" in the nav
1. see you account info, then click `edit`
1. change something and click save, see that it saved
